src/cli.js

```diff
@@ -155,5 +176,10 @@ export async function run(configFile, logFile) {
     return;
   }
 
+  if (configurations.toolingConfigurations.sourceControl === 'GitHub') {
+    configurations.projectConfigurations.projectName =
+      validateProjectNameAgainstGithubRepositories(configurations);
```

Comment:
We may need to await here.  Is this intentionally left out?

Reply:
Yes. There is no need for an await here and await here because there is no asynchronous event.

Reply:
I take that back. An await should be here due to the prompt call.